meetup/services/generic_api.py

```diff
@@ -0,0 +1,21 @@
+from django.http import HttpResponse
+import requests
+from meetuppizza.settings import base
+
+class GenericApi():
+  BASE_URL = ''
+
+  def __init__(self, link):
+    self.link = link
+
+  def get_response(self):
```

Comment:
Does this method have to be empty? Can response fetching and parsing logic be abstracted in any way here?